src/main/groovy/wooga/gradle/unity/models/UnityCommandLineOption.groovy

```diff
@@ -126,6 +133,37 @@ enum UnityCommandLineOption {
      */
     runTests("-runTests"),
 
+    /**
+     * A semicolon-separated list of test categories to include in the run. A semi-colon separated list
+     * should be formatted as a string enclosed in quotation marks, e.g. testCategory "firstCategory;secondCategory".
+     * If using both testFilter and testCategory, then only tests that match both are run. This argument supports negation using '!'.
+     * If using '!MyCategory' then no tests with the 'MyCategory' category will be included in the run.
+     */
+    testCategory("-testCategory", true),
+
+    /**
+     * A semicolon-separated list of test names to run, or a regular expression pattern
+     * to match tests by their full name. A semi-colon separated list should be formatted
+     * as a string enclosed in quotation marks, e.g. testFilter "Low;Medium".
+     * This argument supports negation using '!'. If using the test filter '!MyNamespace.Something.MyTest', then all tests except that test will be run.
+     */
+    testFilter("-testFilter", true),
+
+    /**
+     * The platform to run tests on. Accepted values:
+     * - EditMode: Edit Mode tests. Equivalent to running tests from the EditMode tab of the Test Runner window.
+     * - PlayMode: Play Mode tests that run in the Editor. Equivalent to running tests from the PlayMode tab of the Test Runner window.
+     * - Any value from the BuildTarget enum: Play Mode tests that run on a player built for the specified platform. Equivalent to using the Run all tests (<target_platform>) dropdown in the PlayMode tab of the Test Runner window.
+     * Note: If no value is specified for this argument, tests run in Edit Mode.
+     */
+    testPlatform( "-testPlatform", true),
+
+    /**
+     * A semicolon-separated list of test assemblies to include in the run. A semi-colon separated list
+     * should be formatted as a string enclosed in quotation marks, e.g. assemblyNames "firstAssembly;secondAssembly"
```

Comment:
could you please adjust the line length? It is well longer than ~125 charcters per line

Reply:
sure